criu/net.c

```diff
@@ -3053,6 +3054,54 @@ static int iptables_restore(bool ipv6, char *buf, int size)
 	return ret;
 }
 
+static inline int nftables_lock_network_internal(void)
+{
+#if defined(CONFIG_HAS_NFTABLES_LIB_API_0) || defined(CONFIG_HAS_NFTABLES_LIB_API_1)
+	struct nft_ctx *nft;
+	int ret = 0;
+
+	nft = nft_ctx_new(NFT_CTX_DEFAULT);
+	if (!nft)
+		return -1;
+
+    if (NFT_RUN_CMD(nft, "add table inet CRIU")) {
+		ret = -1;
+        goto free;
+    }
+
+    if (NFT_RUN_CMD(nft, "add chain inet CRIU output { type filter hook output priority 0; policy drop; }")) {
+		ret = -1;
+        goto err;
+    }
+
+    if (NFT_RUN_CMD(nft, "add rule inet CRIU output meta mark " __stringify(SOCCR_MARK) " accept")) {
+		ret = -1;
+        goto err;
+    }
+
+    if (NFT_RUN_CMD(nft, "add chain inet CRIU input { type filter hook input priority 0; policy drop; }")) {
+		ret = -1;
+        goto err;
+    }
+
+    if (NFT_RUN_CMD(nft, "add rule inet CRIU input meta mark " __stringify(SOCCR_MARK) " accept")) {
+		ret = -1;
+        goto err;
+    }
+
+    goto free;
+
+err:
+    NFT_RUN_CMD(nft, "delete table inet CRIU");
+free:
```

Comment:
It would be better to use a different name here because `free` is a function defined in `stdlib.h`. For example:
```diff
-free:
+out:
```